processing/src/main/java/io/druid/query/topn/BaseTopNAlgorithm.java

```diff
@@ -284,7 +291,8 @@ public void keepOnlyN(int n)
       keepOnlyN = n;
     }
 
-    protected Pair<Integer, Integer> computeStartEnd(int cardinality)
+    @VisibleForTesting
```

Comment:
this isn't used in any tests, does it need the expanded scope?

Reply:
Oops, forgot to add test file 😜

Reply:
Oh, nevermind, it's there just diff hidden https://github.com/druid-io/druid/pull/5815/files#diff-6e686ab8ec3a73aeb64ae0583b87af0bR104